public/js/index.js

```diff
@@ -66,6 +66,38 @@ function getYoutubeFeed(playlistid, results, htmlid)
   });
 }
 
+const isToday = () => {
+  // 12 feb 2020
+  someDate = new Date('2020-02-12')
+  let today = new Date();
+  today = someDate.getDate() == today.getDate() &&
+    someDate.getMonth() == today.getMonth() &&
+    someDate.getFullYear() == today.getFullYear();
+  return today;
+};
+
+// ISTORN 2020 countdown
+function istorn2020Counter() {
+  if (isISTORN2020) {
```

Comment:
erm, either this needs to be defined or imported? Codcacy caught it.

Reply:
It's defined in `index.tmpl`, but I wonder what it does on other pages - evaluate to undefined or error out?

Reply:
Console testing makes me think it errors. Will change.